pontis-ui/src/components/Publisher/PublisherCard.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+/**
+ *
+ * @param {React.SVGProps<SVGSVGElement>} props
+ * @return {JSX.IntrinsicElements} an svg element
+ */
+function QuoteIcon(props) {
+  return (
+    <svg aria-hidden="true" width={105} height={78} {...props}>
```

Comment:
Same inline svg question here.
These are also the quotation marks, but they look slightly different from the Testimonial one.

Reply:
Why do we need two different quotation marks? Would prefer to extract into a component and pick one